doc/classes/NavigationAgent3D.xml

```diff
@@ -16,6 +16,13 @@
 				Returns the distance to the target location, using the agent's global position. The user must set the target location with [method set_target_location] in order for this to be accurate.
 			</description>
 		</method>
+		<method name="force_process_avoidance">
+			<return type="Vector3" />
+			<param index="0" name="delta" type="float" />
+			<description>
+				Performs collision avoidance immediately for this agent with the provided [param delta]. This function runs slow and should only be used as a last resort to solve specific edge cases.
```

Comment:
Might also be worth mentioning in the documentation that other agents will have whatever position and velocity they last had.  If `force_process_avoidance` is called in the middle of agents updating their velocities, this agent will not see those updates.

Reply:
Good point, It will calculate the avoidance with the number of agent's and their positions and velocities from the last update frame. The RVO KdTree has not yet rebuild with new added agent's or cleaned up removed agent's and all the position and velocity changes are still in the command queue.